### PR TITLE
Add last_cached_at metadata column for existing shards

### PR DESCRIPTION
**Pull request by khalidharun**
It was discovered that existing shards that don't have a `last_cached_at` column (ie. created before cachemeifyoucan version 0.2.3.4) doesn't work correctly with `safe_columns`.  That issue is now addressed in this PR.

### CODE REVIEW

**Comment by khalidharun** [comment]
@robertzk @kirillseva `dpkg` is erring.  Can you take a look at why CI is broken?

**Comment by kirillseva** [comment]
Nice tests man!

**Comment by khalidharun** [comment]
thanks @kirillseva

**Comment by iveksl2** [comment]
Nice tests

File: tests/testthat/test-data_integrity.R [hunk 1]
```diff
@@ -16,12 +16,14 @@ describe("data integrity", {
...
-  test_that('it crashes when trying to expand a table on new column when safe_columns is TRUE', {
-    dbtest::with_test_db({
-      lapply(dbListTables(test_con), function(t) dbRemoveTable(test_con, t))
-      cached_fcn <- cache(batch_data, key = c(key = "id"), c("model_version", "type"), con = test_con, prefix = prefix, safe_columns = TRUE)
-      cached_fcn(key = 5:1,  model_version, type)
-      expect_error(cached_fcn(key = 1:10, model_version, type, add_column = TRUE))
+  describe("when safe_column is TRUE", {
+    test_that('it crashes when trying to expand a table on new column', {
+      dbtest::with_test_db({
+        lapply(dbListTables(test_con), function(t) dbRemoveTable(test_con, t))
```
Comment (kirillseva) [inline tests/testthat/test-data_integrity.R]: I don't think you need to do this anymore... @peterhurford ?

Comment (khalidharun) [inline tests/testthat/test-data_integrity.R]: @kirillseva Just did a little test. Tables persist between runs.  So this line is still needed.

Comment (peterhurford) [inline tests/testthat/test-data_integrity.R]: No you have to do this using `with_test_db`.  But `dbtest::db_test_that` would do this automatically.

Comment (khalidharun) [inline tests/testthat/test-data_integrity.R]: @peterhurford oh woah.. Nice!  That's really helpful.  Thanks for clearing it up.

**Comment by kirillseva** [comment]
@khalidharun backmerge again

**Comment by khalidharun** [comment]
@kirillseva is there rebuild button for Travis? It hit a github error again.

**Comment by khalidharun** [comment]
@robertzk ready to go

**Pull request merged**